CHANGELOG.rst

```diff
@@ -22,6 +22,7 @@ Features
 * :pr:`125`: Created :file:`setup.cfg` for pytest and tox
 * :gh:`126` (:pr:`127`): Added target for documentation in :file:`tox.ini`
 * :gh:`142` (:pr:`143`): Improved usage section
+* :gh:`144` (:pr:`156`): Added replace function
```

Comment:
Maybe `replace` for syntax highlighting

Reply:
I have rephrased the line to this:

```rst
* :gh:`144` (:pr:`156`): Added :func:`semver.replace` and :func:`semver.VersionInfo.replace`
  functions
```

This has the advantage, that the reader can click on the referenced functions.

Reply:
Very good idea